app/commands/v2/put_content.rb

```diff
@@ -53,12 +54,19 @@ def prepare_content_with_base_path
       def update_content_dependencies(edition)
         create_redirect
         access_limit(edition)
-        update_last_edited_at(edition, payload[:last_edited_at])
         ChangeNote.create_from_edition(payload, edition)
         create_links(edition)
         Action.create_put_content_action(edition, document.locale, event)
       end
 
+      def set_timestamps(edition)
+        if previously_published_edition.is_a?(PreviouslyPublishedItem)
```

Comment:
ooh this looks like a confusing gotcha! - not your fault of course.

Reply:
Yeah - I really don't like how PreviouslyPublishedItem works :(

Reply:
Yeah it's quite confusing as we don't make use of that pattern much so you'd think by the naming it'd be a (item || nil) value. 

If you've not come across this approach before this is a write up of the pattern: https://medium.com/@franzejr/null-object-pattern-in-ruby-12a809bc75cc

Reply:
This article makes it clearer to me 👍